src/pages/index.js

```diff
@@ -1,3 +1,6 @@
-import {LandingPage} from "../pages/landing-page/LandingPage"
+import {ProductListing} from "../pages/product-listing/ProductListing"
+import {Cart} from "../pages/cart/Cart"
+
+export {ProductListing}
+export {Cart}
```

Comment:
You can export these without importing 
`export { componentName } from "path/to/component"`